android/assets/jsons/Civ V - Gods & Kings/GlobalUniques.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Global uniques", 
+    "uniques": [
+        "[-75]% growth [in all cities] <when between [-10] and [0] Happiness>",
+        "Nullifies [Food] [in all cities] <when below [-10] Happiness>",
```

Comment:
Nullifies Growth, not food, no? Nullifies food means everyone starves. And THAT means that we can just use the "[-100]% growth [in all cities] <when below [-10] Happiness>"

Reply:
That was my first plan as well, but then I discovered that if you have multiple growth bonuses (such as from landed elite), the other ones still provide food, which is not what you want. I will change it to nullify growth, though